lib/CompilerManager.js

```diff
@@ -147,7 +147,7 @@ class CompilerManager {
      */
     getAllCompilerInfo() {
         const result = { compilers: {}, leastUsedCompiler: null };
-        Object.entries(this.activeCompilers).forEach(([name, compiler]) => {
+        Object.entries(this.activeCompilers).forEach(([name, _compiler]) => {
```

Comment:
I made this change because Visual Studio said I didn't use `compiler` anywhere in that file, and I was kind of annoyed by the message 😅 , I can remove it if you want.